sonarlint/sonarlint.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>sonarlint</id>
+    <title>SonarLint VSIX Extension</title>
+    <version>2.2.1</version>
+    <authors>SonarSource (SonarSource SA)</authors>
+    <owners>IOZ AG</owners>
+    <projectUrl>http://vs.sonarlint.org/</projectUrl>
+    <packageSourceUrl>https://github.com/iozag/chocolatey-packages/tree/master/git-lfs-install</packageSourceUrl>
+    <licenseUrl>http://opensource.org/licenses/MIT</licenseUrl>
```

Comment:
At least the analyzer is LGPL: https://github.com/SonarSource-VisualStudio/sonaranalyzer-csharp/blob/master/LICENSE